ui/components/app/add-network/add-network.js

```diff
@@ -63,28 +69,73 @@ const AddNetwork = ({
         </Typography>
         {nets.map((item, index) => (
           <Box
             key={index}
             display={DISPLAY.FLEX}
             alignItems={ALIGN_ITEMS.CENTER}
+            justifyContent={JUSTIFY_CONTENT.SPACE_BETWEEN}
             marginBottom={6}
           >
-            <img
-              className="add-network__token-image"
-              src={item?.rpcPrefs?.imageUrl}
-              alt={t('logo', [item.ticker])}
-            />
-            <Typography variant={TYPOGRAPHY.H7} color={COLORS.TEXT_DEFAULT}>
-              {item.ticker}
-            </Typography>
-            <i
-              className="fa fa-plus add-network__add-icon"
-              onClick={onAddNetworkClick}
-              title={`${t('add')} ${item.ticker}`}
-            />
+            <Box display={DISPLAY.FLEX} alignItems={ALIGN_ITEMS.CENTER}>
+              <IconBorder size={24}>
+                <IconWithFallback
+                  icon={item.rpcPrefs.imageUrl}
+                  name={item.nickname}
+                  size={24}
+                />
+              </IconBorder>
+              <Typography
+                variant={TYPOGRAPHY.H7}
+                color={COLORS.TEXT_DEFAULT}
+                fontWeight={FONT_WEIGHT.BOLD}
+                boxProps={{ marginLeft: 2 }}
+              >
+                {item.nickname}
+              </Typography>
+            </Box>
+            <Box display={DISPLAY.FLEX} alignItems={ALIGN_ITEMS.CENTER}>
+              {
+                // Warning for the networks that doesn't use infura.io as the RPC
+                !infuraRegex.test(item.rpcUrl) && (
+                  <Tooltip
+                    className="add-network__warning-tooltip"
+                    position="top"
+                    interactive
+                    html={
+                      <Box margin={3} className="add-network__warning-tooltip">
+                        {t('addNetworkTooltipWarning', [
+                          <a
+                            key="zendesk_page_link"
+                            href="https://metamask.zendesk.com/hc/en-us/articles/4417500466971"
+                            rel="noreferrer"
+                            target="_blank"
+                          >
+                            {t('learnMoreUpperCase')}
+                          </a>,
+                        ])}
+                      </Box>
+                    }
+                    trigger="mouseenter"
+                    theme="light"
+                  >
+                    <img
+                      className="add-network__warning-image"
+                      src="images/warning-triangle-grey.svg"
+                      alt={t('warning')}
+                    />
+                  </Tooltip>
+                )
+              }
+              <div
+                className="add-network__add-button"
+                onClick={onAddNetworkClick}
+              >
+                {t('add')}
+              </div>
```

Comment:
Can use the Button component here
```suggestion
              <Button
                type="inline"
                className="add-network__add-button"
                onClick={onAddNetworkClick}
              >
                {t('add')}
              </Button>
```

Reply:
Done